Kick-Starter/Kick-Starter/Teams Information/TeamsInformationViewModel.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  TeamsInformationViewModel.swift
+//  Kick-Starter
+//
+//  Created by Abdullah Nana on 2021/11/15.
+//
+
+import Foundation
+import KickStarterFramework
+
+final class TeamsInformationViewModel {
```

Comment:
You can add some line breaks in this class to make things a little more readable 😄